datafusion/proto/Cargo.toml

```diff
@@ -55,5 +55,6 @@ serde_json = { workspace = true, optional = true }
 
 [dev-dependencies]
 doc-comment = { workspace = true }
+regex = "1.10.3"
```

Comment:
I think add this dependency for test is OK?

Reply:
I think the test works just fine without actually having `regex`. We can just make the example like
```
struct MyRegexUdf {
 pattern: String
}
```
so we don't need to add an extra dependency.

Reply:
Yeah, I think so.
Fixed it.